src/validation/inputValidation.js

```diff
@@ -0,0 +1,30 @@
+const inputValidation = {
+  checkThreeNum(userInput) {
+    return userInput.length == 3 ? true : false;
```

Comment:
```suggestion
    return (userInput.length === 3);
```
true / false를 반환하기 위한 메서드라면 저 조건만 반환해주어도 true / false가 반환됩니다 !